flake8_spellcheck/__init__.py

```diff
@@ -24,7 +24,7 @@ def to_compat_token(tok):
 def detect_case(name):
     if name.startswith("http"):
         return "url"
-    elif "_" in name:
+    elif "_" in name[1:]:
```

Comment:
could you add a comment here explaining why this is done

Reply:
I've added a comment, but also changed the logic a bit.